### PR TITLE
Add translatations

### PR DESCRIPTION
**Pull request by JuanSeBestia**
Add translations of:
+ Readme
+ US & Requirements

Add US-13
Add US-1-1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,53 @@
 # calculator-angular-tdd
 
-Construyendo una aplicación usando la metodología tdd o test driven development, intentando utilizar las mejores prácticas conocidas por los autores [JuanSeBestia](github.com/JuanSeBestia), [Jess casas](https://github.com/jlcasasg) 
+Builiding a app using the metology tdd or test driven development, We have used the best practice that thea autors know ([JuanSeBestia](github.com/JuanSeBestia), [Jess casas](https://github.com/jlcasasg))
```
Comment (jlcasasg) [inline README.md]: Builiding  -> Building
Builiding a app using the metology  -> Building an app using the methodology 
We have used the best practice that thea autors know -> We have used the best practices that the authors know

File: README.md [hunk 2]
```diff
@@ -1,57 +1,53 @@
...
 
+We espect have all mecanisme related of continuos integration and continus delivey CI/CD
```
Comment (jlcasasg) [inline README.md]: espect  -> expect 
We espect have all mecanisme related of -> We expect to have all mechanisms related to ...

File: README.md [hunk 3]
```diff
@@ -1,57 +1,53 @@
...
 
-
-Se espera tener todos los mecanismos de pruebas, integración y despliegue con CI/CD
-
-* La documentación que no requiera gráfico se realizará en .md dentro de este git
+* The domcuntation that not require grahps would made wit MarckDown inside for this git.
```
Comment (jlcasasg) [inline README.md]: the documentation that doesn't require graphs would be made with ... inside this git

File: README.md [hunk 4]
```diff
@@ -1,57 +1,53 @@
...
-
-Se espera tener todos los mecanismos de pruebas, integración y despliegue con CI/CD
-
-* La documentación que no requiera gráfico se realizará en .md dentro de este git
+* The domcuntation that not require grahps would made wit MarckDown inside for this git. 
 
 ## Sprint 1
 
-En el sprint 1 se prentendrá tener en producción la aplicación en alguna nube en producción.
+In sprint 1 we will pretend to have a production app inside a one cloud.
```
Comment (jlcasasg) [inline README.md]: we pretend to*
inside one cloud

File: README.md [hunk 5]
```diff
@@ -1,57 +1,53 @@
...
 
-### Acerca del producto
+### About the project
 
-Es un aplicación web con la funcionalidad de calculadora, dentro de las funciones de la calculadora se incluyen las operaciones de multiplicar, dividir, sumar, restar y tambien soportará operaciones con decimales.
-
-Se basará sobre una apliacación creada en otra tecnología [JuanSeBestia/react-calculator](https://github.com/JuanSeBestia/react-calculator), diponible en linea aquí FALTA_LINK
+This proyect is inspired in other proyect maded in other framework [JuanSeBestia/react-calculator](https://github.com/JuanSeBestia/react-calculator), see the onlie demo in [DEMO REACT CALCULATOR](https://juansebestia-react-calculator.firebaseapp.com/)
```
Comment (jlcasasg) [inline README.md]: inspired on other project  made

File: README.md [hunk 6]
```diff
@@ -1,57 +1,53 @@
...
 
-Hisotrias y requerimientos de la apliacación [Requeriments_UserStories.md](Requeriments_UserStories.md)
+Histories and requirements of this app: [Requeriments_UserStories.md](wiki/Requeriments_UserStories.md)
 
 
 ## Sprint 2
 
-En el sprint 2 se presntará el mismo producto pero con una funcionalidad adicional de un back, ambos viviendo en la nube.
-
+In sprint 2 we add a one functionaly of the same producto, the new functionality would have had a backend service. The FrontEnd and Backend will coexist in the seame cloud.
```
Comment (jlcasasg) [inline README.md]: we add one functionality of the same product, .... functionality would have a backend...

File: README.md [hunk 7]
```diff
@@ -1,57 +1,53 @@
...
 
-## Acerca de las tenologías
+## About of tecnologies
```
Comment (jlcasasg) [inline README.md]: About Technologies

File: README.md [hunk 8]
```diff
@@ -1,57 +1,53 @@
...
 
-Se emplearán tecnolgías en los que los autores oportunidades de aprender, se utilizará la estrategia de git Feature Branch
+We will use tecnologies wich autors have the oportunities to learn, We use the strategy of git called Git Feture Branch.
```
Comment (jlcasasg) [inline README.md]: technologies, which, authors would have the oportunities ... we will use the git strategy called Git Feature

File: README.md [hunk 9]
```diff
@@ -1,57 +1,53 @@
...
 
-### Desarrollo
+### Development
 
-[Jess casas](https://github.com/jlcasasg) encargada del desarrollo a especificado utilizar Angular & Ionic para desarrollar la aplicación
+[Jess casas](https://github.com/jlcasasg) is encarged of development, she especifed the tecnology to depelopment this app is Angular & Ionic
```
Comment (jlcasasg) [inline README.md]: Jess is in charge of ...  she specified the technology

File: README.md [hunk 10]
```diff
@@ -1,57 +1,53 @@
...
 
-#### Desarrollo: Pruebas Unitarias
+#### Development: Unit test
 
-[Jess casas](https://github.com/jlcasasg) POR DEFINIR
+[Jess casas](https://github.com/jlcasasg) would have used Jest and Enzyme
```
Comment (jlcasasg) [inline README.md]: will use Jest and Enzyme

File: README.md [hunk 11]
```diff
@@ -1,57 +1,53 @@
...
 
 ### Despliegue
```
Comment (jlcasasg) [inline README.md]: Deployment

File: README.md [hunk 12]
```diff
@@ -1,57 +1,53 @@
...
 
-[JuanSeBestia](github.com/JuanSeBestia) Utilizará los servicios de Azure, PENDIENTE POR DEFINIR LOS SERVICIOS ESPECIFICOS
+[JuanSeBestia](github.com/JuanSeBestia) would use the Azure Services (PENDENT TO DEFINE)
```
Comment (jlcasasg) [inline README.md]: would use Azure Services (Pending to define)

File: README.md [hunk 13]
```diff
@@ -1,57 +1,53 @@
...
 
 ### CI/CD
 
-[JuanSeBestia](github.com/JuanSeBestia) PENDIENTE POR DEFINIR
+[JuanSeBestia](github.com/JuanSeBestia) (PENDENT TO DEFINE)
```
Comment (jlcasasg) [inline README.md]: Pending to define

File: README.md [hunk 14]
```diff
@@ -1,57 +1,53 @@
...
 
-## Gestion del proyecto
+## Project managnament
```
Comment (jlcasasg) [inline README.md]: managment*

File: README.md [hunk 15]
```diff
@@ -1,57 +1,53 @@
...
 
-Se tendrán reuniones presenciales o enlinea para discutir los detalles del proyecto, Actuando [JuanSeBestia](github.com/JuanSeBestia) como Product Owner y [Jess casas](https://github.com/jlcasasg) Lider Tecnino y Desarrollador
+The autors will have met in cafe, transport public or Google Meet to discut topics arround if proyect,
```
Comment (jlcasasg) [inline README.md]: The authors will met in cafes, public transport or Google Meet to discuss topics about project

File: README.md [hunk 16]
```diff
@@ -1,57 +1,53 @@
...
 
-[JuanSeBestia](github.com/JuanSeBestia) PENDIENTE POR DEFINIR
+[JuanSeBestia](github.com/JuanSeBestia) (PENDENT TO DEFINE)
 
-## Gestion del proyecto
+## Project managnament
 
-Se tendrán reuniones presenciales o enlinea para discutir los detalles del proyecto, Actuando [JuanSeBestia](github.com/JuanSeBestia) como Product Owner y [Jess casas](https://github.com/jlcasasg) Lider Tecnino y Desarrollador
+The autors will have met in cafe, transport public or Google Meet to discut topics arround if proyect, 
+Se tendrán reuniones presenciales o enlinea para discutir los detalles del proyecto, Acting [JuanSeBestia](github.com/JuanSeBestia) as a Product Owner and [Jess casas](https://github.com/jlcasasg) as a Lead Tech and Developer
```
Comment (jlcasasg) [inline README.md]: They will have personal or online meetings to discuss the project details

File: README.md [hunk 17]
```diff
@@ -1,57 +1,53 @@
...
 
-## Sistema de gestion
+## Management system
 
-Se usará [GitHub Proyects](https://github.com/JuanSeBestia/calculator-angular-tdd/projects) con `Automated Kanban with reviews` para registrar los procesos del los sprints, se utilizarán lo issues de GitHub para reporat bugs
+We going to use [GitHub Proyects](https://github.com/JuanSeBestia/calculator-angular-tdd/projects) with `Automated Kanban with reviews` to register all process inside of sprint, we use the issues of GitHub to reports bugs and new functionalities.
```
Comment (jlcasasg) [inline README.md]: we are going to use ..

File: wiki/Requeriments_UserStories.md [hunk 18]
```diff
@@ -0,0 +1,49 @@
+# Requeriments & User Stories of calculator-angular-tdd
+## Sprint 1
+
+### 1. Funcional requirements
```
Comment (jlcasasg) [inline wiki/Requeriments_UserStories.md]: Functional Requirements

File: wiki/Requeriments_UserStories.md [hunk 19]
```diff
@@ -0,0 +1,49 @@
+# Requeriments & User Stories of calculator-angular-tdd
+## Sprint 1
+
+### 1. Funcional requirements 
+
+  1. The user must be able to use a calculator with the functionalities of the operations of multiplying, dividing, adding, subtracting.
```
Comment (jlcasasg) [inline wiki/Requeriments_UserStories.md]: User must be able

File: wiki/Requeriments_UserStories.md [hunk 20]
```diff
@@ -0,0 +1,49 @@
...
+
+  2. The application must support operations between decimals
+  
+  3. Calculator values must be able to be cleaned
+  
+  4. When the value of the calculator is 666 an image of the devil must be displayed
+  
+  5. When the value of the calculator is 42 should be shown to God
```
Comment (jlcasasg) [inline wiki/Requeriments_UserStories.md]: When the value of the calculator is 42 should be shown to God

 When the value of the calculator is 42 a God image must be shown

File: wiki/Requeriments_UserStories.md [hunk 21]
```diff
@@ -0,0 +1,49 @@
...
+
+### 2. No funcional requirements
```
Comment (jlcasasg) [inline wiki/Requeriments_UserStories.md]: No functional requirements

File: wiki/Requeriments_UserStories.md [hunk 22]
```diff
@@ -0,0 +1,49 @@
...
+
+  1. The background of the application must have a background of a modern concept (the developer must propose some)
+
+  2. The application must at least look the same or more modern than the example given
```
Comment (jlcasasg) [inline wiki/Requeriments_UserStories.md]: must look at least the same or more modern than...

File: wiki/Requeriments_UserStories.md [hunk 23]
```diff
@@ -0,0 +1,49 @@
...
+
+### 3. Historias de usuario
```
Comment (jlcasasg) [inline wiki/Requeriments_UserStories.md]: User Stories

File: wiki/Requeriments_UserStories.md [hunk 24]
```diff
@@ -0,0 +1,49 @@
...
+  1. I as a user want to see the value that the calculator currently has, to know the result of my options
```
Comment (jlcasasg) [inline wiki/Requeriments_UserStories.md]: result of my operations...

File: wiki/Requeriments_UserStories.md [hunk 25]
```diff
@@ -0,0 +1,49 @@
...
+    
+      1. The initial value is "0"
+  
+  2. I as a user must be able to click on buttons to enter number 1-9
+  
+  3. I as a user want to press a button to demarcate the decimal septation of the number
```
Comment (jlcasasg) [inline wiki/Requeriments_UserStories.md]: the decimal separation of the current number

File: wiki/Requeriments_UserStories.md [hunk 26]
```diff
@@ -0,0 +1,49 @@
...
+
+  4. I, as a user, want to press a button to indicate that I want to use the multiply functionality
```
Comment (jlcasasg) [inline wiki/Requeriments_UserStories.md]: Remove commas

File: wiki/Requeriments_UserStories.md [hunk 27]
```diff
@@ -0,0 +1,49 @@
...
+
+  5. I as a user want to press a button to indicate that I want to use the functionality of the division
```
Comment (jlcasasg) [inline wiki/Requeriments_UserStories.md]: division functionality

File: wiki/Requeriments_UserStories.md [hunk 28]
```diff
@@ -0,0 +1,49 @@
...
+
+  6. I as a user want to press a button to indicate that I want to use the subtraction functionality
+
+  7. I as a user want to press a button to indicate that I want to use the sum functionality
+
+  8. I as a user want to press a button to indicate that I want to see the result of the operation
+
+  9. I as a user want to press a button to indicate that I want to clean the calculator of the operations performed, to perform new operations
+
+  10. I as a user want to see the image of a demon at the time the calculator has the value 666
```
Comment (jlcasasg) [inline wiki/Requeriments_UserStories.md]: please redact this in other way,
I as a user want to see a demon image when the result of the operation is 666

File: wiki/Requeriments_UserStories.md [hunk 29]
```diff
@@ -0,0 +1,49 @@
...
+
+  11. I as a user want to see the image of a god at the time the calculator has the value 42
```
Comment (jlcasasg) [inline wiki/Requeriments_UserStories.md]: same as above:
I as a user want to see a God image when tthe result of the operation has the 42 value

File: wiki/Requeriments_UserStories.md [hunk 30]
```diff
@@ -0,0 +1,49 @@
...
+
+  12. I as the user of the application, wish that the fund contains a modern connection so that I feel like using it and I am pleased to be there
```
Comment (jlcasasg) [inline wiki/Requeriments_UserStories.md]: missed translation!

File: wiki/Requeriments_UserStories.md [hunk 31]
```diff
@@ -0,0 +1,49 @@
...
+
+  13. I as the user when error is happend, I want to see the error inside the display and the erro is would be errased when tap some button. When tap some button in the display would be the previous value after the error
```
Comment (jlcasasg) [inline wiki/Requeriments_UserStories.md]: missed translation!

Comment (JuanSeBestia) [reply wiki/Requeriments_UserStories.md]: I think no

File: wiki/Requeriments_UserStories.md (hunk 30 above)
Comment (JuanSeBestia) [reply wiki/Requeriments_UserStories.md]: Done

File: README.md (hunk 1 above)
Comment (JuanSeBestia) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (JuanSeBestia) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (JuanSeBestia) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (JuanSeBestia) [reply README.md]: Done

File: README.md (hunk 6 above)
Comment (JuanSeBestia) [reply README.md]: Done

File: README.md (hunk 7 above)
Comment (JuanSeBestia) [reply README.md]: Done

File: README.md (hunk 9 above)
Comment (JuanSeBestia) [reply README.md]: Done

File: README.md (hunk 10 above)
Comment (JuanSeBestia) [reply README.md]: Done

File: README.md (hunk 12 above)
Comment (JuanSeBestia) [reply README.md]: Done

File: README.md (hunk 13 above)
Comment (JuanSeBestia) [reply README.md]: Done

File: README.md (hunk 14 above)
Comment (JuanSeBestia) [reply README.md]: Done

File: README.md (hunk 15 above)
Comment (JuanSeBestia) [reply README.md]: Done

File: README.md (hunk 17 above)
Comment (JuanSeBestia) [reply README.md]: Done

File: wiki/Requeriments_UserStories.md (hunk 18 above)
Comment (JuanSeBestia) [reply wiki/Requeriments_UserStories.md]: Done

File: wiki/Requeriments_UserStories.md (hunk 19 above)
Comment (JuanSeBestia) [reply wiki/Requeriments_UserStories.md]: Done

File: wiki/Requeriments_UserStories.md (hunk 20 above)
Comment (JuanSeBestia) [reply wiki/Requeriments_UserStories.md]: Done

File: wiki/Requeriments_UserStories.md (hunk 21 above)
Comment (JuanSeBestia) [reply wiki/Requeriments_UserStories.md]: Done

File: wiki/Requeriments_UserStories.md (hunk 22 above)
Comment (JuanSeBestia) [reply wiki/Requeriments_UserStories.md]: Done

File: wiki/Requeriments_UserStories.md (hunk 23 above)
Comment (JuanSeBestia) [reply wiki/Requeriments_UserStories.md]: Done

File: wiki/Requeriments_UserStories.md (hunk 24 above)
Comment (JuanSeBestia) [reply wiki/Requeriments_UserStories.md]: Done

File: wiki/Requeriments_UserStories.md (hunk 25 above)
Comment (JuanSeBestia) [reply wiki/Requeriments_UserStories.md]: Done

File: wiki/Requeriments_UserStories.md (hunk 26 above)
Comment (JuanSeBestia) [reply wiki/Requeriments_UserStories.md]: Done

File: wiki/Requeriments_UserStories.md (hunk 27 above)
Comment (JuanSeBestia) [reply wiki/Requeriments_UserStories.md]: Done

File: wiki/Requeriments_UserStories.md (hunk 28 above)
Comment (JuanSeBestia) [reply wiki/Requeriments_UserStories.md]: Done

File: wiki/Requeriments_UserStories.md (hunk 30 above)
Comment (JuanSeBestia) [reply wiki/Requeriments_UserStories.md]: Done

File: wiki/Requeriments_UserStories.md (hunk 31 above)
Comment (JuanSeBestia) [reply wiki/Requeriments_UserStories.md]: I try

File: README.md (hunk 4 above)
Comment (JuanSeBestia) [reply README.md]: done

File: README.md (hunk 8 above)
Comment (JuanSeBestia) [reply README.md]: Done

File: README.md (hunk 11 above)
Comment (JuanSeBestia) [reply README.md]: Done

File: README.md (hunk 16 above)
Comment (JuanSeBestia) [reply README.md]: Done

File: wiki/Requeriments_UserStories.md (hunk 29 above)
Comment (JuanSeBestia) [reply wiki/Requeriments_UserStories.md]: Done

**Pull request merged**